src/components/signup-form/SignupForm.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import { Button, Col, Form, Input, Row, Typography } from 'antd';
+import { ParagraphProps } from 'antd/lib/typography/Paragraph';
+import { Link } from 'react-router-dom';
+import { TEXT_GREY } from '../../colors';
+import styled from 'styled-components';
+import useWindowDimensions, {
+  WindowTypes,
+} from '../../components/window-dimensions';
+
+const { Paragraph } = Typography;
+
+const formHalfItemSpan = 11;
+const offsetSpan = 1;
+
+const formLayout = {
+  wrapperCol: { span: 17 },
+};
+
+const Footer: typeof Paragraph = styled(Paragraph)<ParagraphProps>`
+  color: ${TEXT_GREY};
+  line-height: 1.5;
+`;
+
+interface SignupFormProps {
+  readonly onFinish: any;
+}
+
+interface ButtonFooterProps {
+  readonly isMobile: boolean;
+}
+
+const ButtonFooter: React.FC<ButtonFooterProps> = ({ isMobile }) => {
```

Comment:
If this is large enough to be a separate component, it should go in it's own file as well. Personally, I think it makes more sense to include it directly on line 137.